nomad/grapl-core.nomad

```diff
@@ -0,0 +1,428 @@
+                        "alpha",
+                        "--my=localhost:${alpha.value.grpc_private_port}",
+                        "-o", "${alpha.value.id}",
+                        "--zero", "${local.zero_alpha_connect_str}"
+                    ]
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-grpc-private"
+                port = "${alpha.value.grpc_private_port}"
+
+                connect {
+                    sidecar_service {
+                        proxy {
+                            # Connect to the Zero leader
+                            upstreams {
+                                destination_name = "dgraph-zero-leader-grpc-private"
+                                local_bind_port = 5080
+                            }
+
+                            # Connect this Alpha to Zero followers
+                            dynamic "upstreams" {
+                                iterator = zero_follower
+                                for_each = [for zero_follower in local.dgraph_zeros: zero_follower]
+
+                                content {
+                                    destination_name = "dgraph-zero-follower-${zero_follower.value.id}-grpc-private"
+                                    local_bind_port = zero_follower.value.grpc_private_port
+                                }
+                            }
+
+                            # Connect this Alpha to Other Alphas (but not to itself, obviously)
+                            dynamic "upstreams" {
+                                iterator = alpha_peer
+                                for_each = [for alpha_peer in local.dgraph_alphas: alpha_peer if alpha_peer.id != alpha.value.id]
+
+                                content {
+                                    destination_name = "dgraph-alpha-${alpha_peer.value.id}-grpc-private"
+                                    local_bind_port = alpha_peer.value.grpc_private_port
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-grpc-public"
+                port = "${alpha.value.grpc_public_port}"
+
+                connect {
+                    sidecar_service { }
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-http"
+                port = "${alpha.value.http_port}"
+
+                connect {
+                    sidecar_service { }
+                }
+            }
+        }
+    }
+
+    group "grapl-graph-merger" {
+        count = var.graph_mergers
+
+        network {
+            mode = "bridge"
+        }
+
+        task "graph-merger" {
+            driver = "docker"
+
+            config {
+                image = "${var.container_registry}/graph-merger:${var.tag}"
+                args = ["/graph-merger"]
```

Comment:
Is this supposed to be a command?

If so, this should not be required anymore, because the Rust containers all have their respective binaries set as the container entrypoints now. Simply running the container will be the equivalent of running that binary.

Reply:
> Is this supposed to be a command?

Yup! I needed it to actually get the container running (in my testing).

> ..this should not be required anymore..

I'll give it a go! Thanks for the heads up!

Reply:
FYI, this changed in https://github.com/grapl-security/grapl/pull/1039

Reply:
Seemed to work for me! Thanks for the heads up :)